browser/src/control/Control.AutofilterDropdown.js

```diff
@@ -131,6 +131,9 @@ L.Control.AutofilterDropdown = L.Control.extend({
 				top = top - splitPos.y;
 		}
 
+		if (this._map._docLayer.isCalc() && this._map._docLayer.isLayoutRTL())
+			left = this._map._size.x - left;
```

Comment:
Shouldn't we always have braces?

Reply:
Ideally not - so the code is readable ;-)